_pages/learn.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: splash
+permalink: /learn.html
+title: "Python Tutorials & Learning Resources for Scientists"
+classes: flowing
+header:
+  overlay_image: images/header.png
+  overlay_filter: 0.6
+scientists:
+  - image_path: images/pyopensci-people-at-computer.jpg
+    alt: "A pencil sketch of a round table with people sitting around it from different backgrounds working on laptops and also writing together."
+    excerpt: "Our catalog of vetted open source tools makes it easier for scientists to find the trusted tools that they need to develop their open science workflows."
+    title: "Help scientists find the open science tools that they need"
+    url: https://www.pyopensci.org/python-packages.html
+    btn_label: View our growing list of accepted scientific Python packages
+    btn_class: btn--primary
+overview:
+  - excerpt: "pyOpenSci creates resources to help you navigate the Python packaging ecosystem with ease. All of our materials are community developed and go through extensive review."
+development:
+   - image_path: images/pyopensci-learn-community-created.png
+     alt: "alt here "
+     title: "Community Developed Python Tutorials for Scientists"
```

Comment:
```suggestion
     title: "Community-developed Python Tutorials for Scientists"
```

Reply:
I think I'm a bit confused on the difference between the Community-developed and Beginner-friendly sections. in reading through everything, it feels like community-developed is the main header, and gives information about our tutorials, and that beginner-friendly is a subheader and example of community-developed tutorials.

from a UI//UX perspective, if both are being treated as H1s, then I'd expect the Community-developed section to link out to something, and stand on its own (maybe some of the reviews or bug bashes, or a CTA to follow repos and get involved?) otherwise, if beginner-friendly is an H2 subheader of H1, then I think we can extend the purple background and shrink the space between the two, and provide some kind of linking text that makes it clear how the two sections are related to one another.

Reply:
ok great. i don't think we have text on community developed anywhere in the site - YET. so this is our first stab. 

the idea is this section talks about how we have this extensive review process, run bug bashes, and create a collab envt where content is co-created and we are leading that co creation but promoting a healthy environment to build consensus. we could write more.

the following section is just a listing of tutorials that we have now - and they are beginner friendly. so i see these two things are separate (ie both h1).

what do you suggest to make this better / more clear? i think your point is excellent i'd love some feedback on implementation.

Reply:
that makes sense. I think what would make it more clear is to change the headline to "Beginner-friendly" rather than "Community-developed", and then as a sub-section have something along the lines of "All pyOpenSci Python tools are community-developed" with the paragraph about community developed underneath it